documentation/site/themes/tbds/layouts/example/baseof.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="{{ $.Site.Language.Lang }}">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+    <base href="{{ if getenv "CONTEXT" }}{{ cond (eq "production" (getenv "CONTEXT")) (getenv "URL") (getenv "DEPLOY_PRIME_URL") }}{{ else }}{{ $.Site.BaseURL }}{{ end }}">
```

Comment:
Is this `base` element needed?

Reply:
Hmm, I had taken this from the other layout, but now I wonder what it's doing. it looks like it's setting the [base](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) for relative URLs? I've never used that. I wonder if this is some Victor Hugo feature to have a production deploy on a sub-folder? Since we don't use that I wonder if I can just get rid of it.

Reply:
Ok, I can't explain why, but it looks like it needed it for now.